test/views/sections/items/ItemCommodityCodeViewSpec.scala

```diff
@@ -58,7 +58,7 @@ class ItemCommodityCodeViewSpec extends ViewSpecBase with ViewBehaviours {
       )
 
       val testCommodityCode2 = CnCodeInformation(
-        cnCode = "testCnCode2",
+        cnCode = "testCnCodeWine",
```

Comment:
Does this need to change?
```suggestion
        cnCode = testCnCodeWine,
```